packages/client/src/generation/ts-builders/AnyDeclarationBuilder.ts

```diff
@@ -1,5 +1,6 @@
 import { ConstDeclaration } from './ConstDeclaration'
+import { InterfaceDeclaration } from './Interface'
 import { TypeDeclaration } from './TypeDeclaration'
 
-// TODO: enum, class, interface
-export type AnyDeclarationBuilder = TypeDeclaration | ConstDeclaration
+// TODO: enum, class
```

Comment:
TODO: tech debt issue?

Reply:
I think this is fine without one, at the moment we are not generating enums and classes with ts-builder because we had no need to so far. Once we have said need, we'll add those but I won't spend time for implementing it right now.